.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -21,8 +21,8 @@
 
 ## Types of changes
 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Chore (non-breaking change that fixes an issue)
```

Comment:
Chore (refactor or non-breaking change that fixes an issue) maybe? :P

Reply:
or Chore (non-breaking change that fixes an issue or refactor of existing code)

Reply:
tests, refactors and fixes

Reply:
I don't think mentioning non-breaking changes is necessary to be honest 🤷‍♀️